### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#361]
I'm missing in poeditor.com:

- Don't add to list
- Overwrite (but this word is already correctly translated, somewhere else in webtrees)

When overwriting a setting a pop-up window appears "Do you like to overwrite?" (or something similar). That is translated correctly, but there are two buttons and they are not translated:
- Cancel
- Overwrite.

I replaced in the German version all "Startperson" by "Quellenperson" (= source person), for example in 

- Invalid starting individual
- Show starting persons in different color
- Starting persons background color

**Comment by Neriderc** [comment]
'Overwrite' is already translated in POEditor for that dialogue with Cancel/Overwrite. However, I did not correctly load this in the code, which I'll fix soon.

I have now added "Don't add to list" to POEditor. Sorry, I was sure I had done this along with the others but it seems I hadn't.

So just to confirm, we are swapping all "Starting persons" or "Starting individuals" for "Source individuals"? If so I'll go through and update all the references to these terms.

**Comment by hartenthaler** [comment]
I'm not sure about swapping "starting" to "source". I tried it in the German translation and it is possible to do it in a consistent way. But we still have the "stopping" list, so "starting" fits better. In German it is "Startperson" und "Stopperson" and this sounds good, too. I can swap it back in the translation. 

The help text for the "Treatment" option is very clear, so there is no need for changing "starting",

@schuco: what do you think about?

**Comment by Neriderc** [comment]
Reopening as pull request didn't close issue I wanted it to and did close the one I didn't want it to :facepalm:

**Comment by hartenthaler** [comment]
If I see it right: the only open topic for milestone 2.1.17 is:
- keep "starting persons" or swap to "source persons"

I would prefer "starting". Then the original (English) terms could be as they are, but I have to swap back in the German translation.

**Comment by Neriderc** [comment]
It seems we sometimes use the term "starting persons" and sometimes use the term "starting individuals". Should we make this more consistent and always use "starting individuals"? I suggest this option as webtrees refers to "individuals" rather than "persons" so it seems best to use this term.

Other than this, I'll make updates to the Github wiki and will update the changlog file (which is partly done already). Then this release should be ready!

**Comment by hartenthaler** [comment]
"starting individuals" are good. 

I swapped back in the German translation to "Startperson".

In the help text:

> This is our starting point. Choose a person to base the diagram around. You might choose yourself.</p><p>You can choose as many starting individuals as you wish. A tree will be created for each starting individual based on the settings, then the trees are merged to create one tree for the diagram.</p><p>Click the X to remove a record from the list. If two or more records are listed, a "Clear all" option will appear to allow you to remove all records.</p><p>You can also click on an individual in this list to have the diagram in the browser scroll to that person.</p><p>When you next come to the page, the default individual or the last individual you were looking at in webtrees will be loaded. If there was just one saved individual then it will be replaced with the individual provided by webtrees on coming to the page. If there were multiple in the list, then the individual provided by webtrees will be added to the list (if not already in it). So be sure to double-check you are only including individuals you are looking for when first loading the page.

I added at the end of the translation of this text a sentence:

`Dieses standardmäßige Verhalten kann mit Hilfe der Option "Behandlung einer neuen Startperson" geändert werden.
`
 and modified some minor points.

**Comment by Neriderc** [comment]
Oh yes good idea, I've updated that in English as well.

I think we are ready to release 2.1.17. I have pulled the latest version of all the translations, updated the version number in the module code, updated the changelog, updated the wiki. I will soon create the pull request to merge in the changes, then will write release notes and finally update the latest-version file.

This release has been a long time coming! Almost 3 months since the last release, which I think is by far the longest gap since I created this fork. But it's a big release! I'm very excited for all the bug reports we are sure to get :slightly_smiling_face: 

Thanks @hartenthaler and @schuco for testing, reporting bugs, and also for coming up with the ideas for the features and helping to refine them. The new feature to save versions of settings and all the things that this leads to is really quite exciting!
